### PR TITLE
rename dataset for training 

### PR DESCRIPTION
**Pull request by Schirni**
swap to sdo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,126 +10,13 @@ We demonstrate that the available data sets can directly profit from the most re
...
-                         months=list(range(11)))
-# Init validation/plotting datasets
-sdo_valid = SDODataset(sdo_data_path,
-                       resolution=2048, patch_shape=(256, 256),
-                       months=[11, 12], limit=100)
-soho_valid = SOHODataset(soho_data_path,
-                         resolution=1024, patch_shape=(128, 128),
-                         months=[11, 12], limit=100)
+## Usage
+To get started with the ITI tool we recommend to play around with the [example](https://iti-documentation.readthedocs.io/en/latest/generated/gallery/index.html) notebooks.
```
Comment (RobertJaro) [inline README.md:19]: play around is bad wording.
--> We provide an example gallery that can be used as a starting point.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,7 @@
 ![](images/HinodeEnhanced_v2.jpg)
-# Instrument to Instrument Translation for Solar Observations
+# Instrument to Instrument Translation (ITI) for Solar Observations
 
-# [Paper](#paper) --- [Usage](#usage) --- [Framework](#framework) 
-
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/RobertJaro/InstrumentToInstrument/blob/master/examples/ITI_translation.ipynb)
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1nwXqT2SI1xqO1Vz9aYQeAadzriL7UFVM#scrollTo=k_WqPbLGYmea) [![Documentation Status](https://readthedocs.org/projects/iti-documentation/badge/?version=latest)](https://iti-documentation.readthedocs.io/en/latest/?badge=latest)
```
Comment (RobertJaro) [inline README.md:5]: It would be better to have the notebooks in github in order to have them archived and version tracked.

**Review by RobertJaro — CHANGES_REQUESTED** [review]
please move the notebooks to github and adjust wording

File: README.md (hunk 1 above)
Comment (Schirni) [reply README.md]: added example notebook to examples folder.

Comment (Schirni) [reply README.md]: changed wording.

File: README.md (hunk 2 above)
Comment (Schirni) [reply README.md:5]: added notebook to examples folder

**Comment by Schirni** [comment]
- updated readme 
- adjusted wording
- added notebook to github

**Pull request merged**